scrapy/extensions/feedexport.py

```diff
@@ -420,15 +426,12 @@ def item_scraped(self, item, spider):
             slot.start_exporting()
             slot.exporter.export_item(item)
             slot.itemcount += 1
-            # create new slot for each slot with itemcount == FEED_EXPORT_BATCH_ITEM_COUNT and close the old one
-            if (
-                self.feeds[slot.uri_template]['batch_item_count']
-                and slot.itemcount >= self.feeds[slot.uri_template]['batch_item_count']
-            ):
+            self.batches[slot.uri_template].update()
+            # create new slot if slot's batch should trigger and close the old one
```

Comment:
This comment does not seem to add much information to what is already clear from the code, I think.